lib/elixir_sense/core/introspection.ex

```diff
@@ -443,6 +447,8 @@ defmodule ElixirSense.Core.Introspection do
     "**Delegates to**\n#{inspect(m)}.#{f}/#{a}"
   end
 
+  defp get_metadata_entry_md({_, _}), do: nil
```

Comment:
I’d rather see a warning or a generic case similar to other. Returning nil will prevent us from seeing that we are not handling something.

Reply:
Sure, we could do something like
```suggestion
  defp get_metadata_entry_md({key, value}) do
    "**#{key}** (warning: unhandled)\n#{value}"
  end
```